api/http/api_deployments.go

```diff
@@ -1066,6 +1066,45 @@ func (d *DeploymentsApiHandlers) GetDeploymentStats(w rest.ResponseWriter, r *re
 	d.view.RenderSuccessGet(w, stats)
 }
 
+func (d *DeploymentsApiHandlers) GetDeploymentsStats(w rest.ResponseWriter, r *rest.Request) {
+
+	ctx := r.Context()
+	l := requestlog.GetRequestLogger(r)
+
+	ids := model.DeploymentIDs{}
+	if err := r.DecodeJsonPayload(&ids); err != nil {
+		d.view.RenderError(w, r, err, http.StatusBadRequest, l)
+		return
+	}
+
+	if len(ids.IDs) == 0 {
+		w.WriteHeader(http.StatusOK)
+		_ = w.WriteJson([]byte(`{}`))
+		return
+	}
+
+	for _, id := range ids.IDs {
+		if !govalidator.IsUUID(id) {
+			d.view.RenderError(w, r, ErrIDNotUUID, http.StatusBadRequest, l)
+			return
+		}
+	}
+
```

Comment:
There's no limit on the number of items you can push into this schema. Can we put a limit on at most 100 items?

Reply:
https://github.com/mendersoftware/deployments/pull/666/commits/014e8fe0224138b6db004290337063868fb26747